CHANGELOG.md

```diff
@@ -78,7 +56,72 @@ If you're not using Nunjucks macros, then you should:
 
 Check your changes against the [header example in the Design System](https://design-system.service.gov.uk/components/header/#header-with-service-name-and-navigation).
 
-This change was introduced in [pull request #2427: Improve the screen reader experience for the header menu](https://github.com/alphagov/govuk-frontend/pull/2427).
+These changes were introduced in [pull request #2427: Improve the screen reader experience for the header menu](https://github.com/alphagov/govuk-frontend/pull/2427) and [pull request #2471: Add missing instructions for header to changelog](https://github.com/alphagov/govuk-frontend/pull/2471).
```

Comment:
@36degrees Is this the right place to mention #2471? Added it here, following [your comment in the Google Doc re making sure we don't lose changes](https://docs.google.com/document/d/1atVBJI2ETBskZay_coZJxEsL1T4BOgoKY9BKszyr4Bo/edit?disco=AAAAS2EIacI).